src/components/essences/essences-toolbar.tsx

```
@@ -1,97 +1,145 @@
+          ))}
+          <li />
+          <li>
+            <button
+              className="btn btn-ghost"
+              disabled={queryStates.rarities.length === 0}
+              onClick={() =>
+                setQueryStates((prev) => ({ ...prev, rarities: [] }), {
+                  startTransition: raritiesStartTransition,
+                })
+              }
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)